src/platform/site-wide/user-nav/components/PageNotFound.jsx

```diff
@@ -1,6 +1,9 @@
 import React, { useEffect } from 'react';
-import recordEventFn from '@department-of-veterans-affairs/platform-monitoring/record-event';
 import PropTypes from 'prop-types';
+// Cypress does not like @ imports, so import record-event with a path
```

Comment:
I had issues with `@` imports (and instead used `~`) from platform monitoring in my downtime alerts PR. I didn't have time to figure out if the issue was with the code or my understanding. I got as far as: maybe babel (and/or eslint?) likes things one way, cypress another?

Reply:
Cypress is not able to resolve the @ imports. The preference is to always use @